draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1932,8 +1931,7 @@ consisting of:
 ## Signature Algorithms    {#signature_algorithms}
 
 IANA is asked to establish a registry of signature algorithm values, named
-"Signature algorithm numbers for Certificate Transparency logs" initially
-consisting of:
+"CT Signature Algorithms" initially consisting of:
```

Comment:
There needs to be a comma before "initially".

I think "s/ initially consisting/, that initially consists/" would be better still.